zap/src/main/java/org/zaproxy/zap/extension/search/SearchThread.java

```diff
@@ -270,6 +270,16 @@ private void search() {
                             }
                         }
                     }
+                    if (Type.Tag.equals(reqType) && !pcc.allMatchesProcessed()) {
+                        for (String tag : message.getHistoryRef().getTags()) {
+                            matcher = pattern.matcher(tag);
+                            if (matcher.find()) {
+                                notifyMatchFound(
+                                        currentRecordId, matcher.group(), message, null, 0, 0);
```

Comment:
I'd suggest the name the tag instead of the match, if you do a partial search it only shows part of it, sometimes it's hard to tell what the actual tag is.

Reply:
Ok, I'll switch that shortly.